modules/oadp-installing-dpa-1-3.adoc

```diff
@@ -0,0 +1,385 @@
+<8> Specify a bucket as the backup storage location. If the bucket is not a dedicated bucket for Velero backups, you must specify a prefix.
+<9> Specify a prefix for Velero backups, for example, `velero`, if the bucket is used for multiple purposes.
+<10> Specify a snapshot location, unless you use CSI snapshots or Restic to back up PVs.
+<11> The snapshot location must be in the same region as the PVs.
+endif::[]
+ifdef::installing-oadp-mcg[]
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  name: <dpa_sample>
+  namespace: openshift-adp
+spec:
+  configuration:
+    velero:
+      defaultPlugins:
+        - aws
+        - openshift <1>
+      resourceTimeout: 10m <2>
+    nodeAgent: <3>
+      enable: true <4>
+      uploaderType: kopia <5>
+      podConfig:
+        nodeSelector: <node_selector> <6>
+  backupLocations:
+    - velero:
+        config:
+          profile: "default"
+          region: minio
+          s3Url: <url> <7>
+          insecureSkipTLSVerify: "true"
+          s3ForcePathStyle: "true"
+        provider: {provider}
+        default: true
+        credential:
+          key: cloud
+          name: {credentials} <8>
+        objectStorage:
+          bucket: <bucket_name> <9>
+          prefix: <prefix> <10>
+----
+<1> The `openshift` plugin is mandatory.
+<2> Specify how many minutes to wait for several Velero resources before timeout occurs, such as Velero CRD availability, volumeSnapshot deletion, and backup repository availability. The default is 10m.
+<3> The administrative agent that routes the administrative requests to servers.
+<4> Set this value to `true` if you want to enable `nodeAgent` and perform File System Backup.
+<5> Enter `kopia` or `restic` as your uploader. You cannot change the selection after the installation. The `nodeAgent` deploys a daemon set, which means that the `nodeAgent` pods run on each working node. You can configure File System Backup by adding `spec.defaultVolumesToFsBackup: true` to the `Backup` CR.
+<6> Specify the nodes on which Kopia or Restic are available. By default, Kopia or Restic run on all nodes.
+<7> Specify the URL of the S3 endpoint.
+<8> If you do not specify this value, the default name, `{credentials}`, is used. If you specify a custom name, the custom name is used for the backup location.
+<9> Specify a bucket as the backup storage location. If the bucket is not a dedicated bucket for Velero backups, you must specify a prefix.
+<10> Specify a prefix for Velero backups, for example, `velero`, if the bucket is used for multiple purposes.
+endif::[]
+ifdef::installing-oadp-ocs[]
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  name: <dpa_sample>
+  namespace: openshift-adp
```

Comment:
Update NS here too?

Reply:
@kaovilai 

If I change the NS in the Installation DPA, does it require changing the NS throughout the documentation?

Reply:
ya.. so the namespace OADP installs into by default is openshift-adp, however the namespace name can be configured to be anything.  For documentation purposes IMHO coming up with a standard across the OADP docs seems like the right move.

We could use the default name openshift-adp everywhere if there is a corresponding note the namespace is configurable.  

The important thing would be:
1. A documentation standard for the namespace used for OADP in documentation
2. Clear documentation that is referable and cross linked that indicates the namespace name is a variable and for the sake of doc we're using the default 'openshift-adp'

Reply:
+1 to using 'openshift-adp' as default along with a note mentioning that it is configurable.

Reply:
@weshayutin @stillalearner @kaovilai 

<1> The default namespace for OADP is `openshift-adp`. The namespace is a variable and is configurable.

Reply:
Works for me!.
Thanks!

Reply:
Looks good.